### DISCUSSION TITLE
Integrated Controls for Mobile

### DISCUSSION THREAD

**Discussion by blitzve0** [idiotWu/smooth-scrollbar#395] [Ideas]
When declaring smooth scrollbar, it would be so much better i we can define something like mobile: false, mobileDamping: 0.1.or so some of mobile controls so that we can use it on mobiles too.

**Comment by dolphin-wood** [comment]
Did you mean you need different damping values between desktops and mobiles?

**Reply by blitzve0** [reply]
Yes and maybe a little more mobile friendly config

**Reply by dolphin-wood** [reply]
Thought it could be done with:

```js
damping: isMobile ? 0.5 : 0.1
```

**Reply by dolphin-wood** [reply]
Related discussions: #404
